app/modules/wifi.c

```diff
@@ -730,45 +788,10 @@ static int wifi_station_setauto( lua_State* L )
   a = luaL_checkinteger( L, 1 );
   luaL_argcheck(L, ( a == 0 || a == 1 ), 1, "0 or 1");
   wifi_station_set_auto_connect(a);
-  return 0;  
+  return 0;
```

Comment:
I'll help with that. Right before or after a master drop just run wtf.py on the code repo and make a commit. 

Using the gist below just run `wtf-find -0 .` to get a summary of issues. `wtf-find -i .` to fix line endings, extra spaces, and EOF issues.

https://gist.github.com/karrots/49340d04c56b9d4d3639